x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/actions/connector_types/bedrock.ts

```diff
@@ -432,38 +425,6 @@ export default function bedrockTest({ getService }: FtrProviderContext) {
             });
           });
 
-          it('should invoke stream with assistant AI body argument formatted to bedrock expectations', async () => {
-            await new Promise<void>((resolve, reject) => {
```

Comment:
See: https://github.com/elastic/kibana/issues/188467

Reply:
@ymao1 would you be willing to review this PR? I think the only responseops changes are removing this test. Thanks!